clang/lib/DPCT/ThrustAPIMigration.cpp

```diff
@@ -19,26 +19,45 @@ using namespace clang::ast_matchers;
 void ThrustAPIRule::registerMatcher(ast_matchers::MatchFinder &MF) {
   // API register
   auto functionName = [&]() { return hasAnyName("on"); };
-  // Both THRUST_200302___CUDA_ARCH_LIST___NS and __4 are newly imported inline
+  // THRUST_200302___CUDA_ARCH_LIST___NS is newly imported inline
   // namespace by thrust library in CUDA header file 12.4.
+  // THRUST_200400___CUDA_ARCH_LIST___NS is newly imported inline
```

Comment:
Please consider opt this match,  in runtime only one value is defined for the namespace.

Reply:
Okay, will implement it in  the next update PR.